### ISSUE TITLE
Can't send content_name and other important properties

### ISSUE THREAD

**Issue by 10xchs** [RivercodeAB/facebook-conversion-api-nextjs#26]
You cannot send `content_name` or most of the properties/parameters listed here:

https://developers.facebook.com/docs/meta-pixel/reference#utilizing_params

**Comment by maxout** [comment]
Yes, that really is a problem. Too bad that not all properties are simply passed
